File: python/resistor-color-duo/resistor_color_duo.py
def value(colors):
    pallete = [
            "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white",
        ]

    return sum([pallete.index(color)*pow(10, 1-index) for index, color in enumerate(colors[:2])])

def value1(colors):
    pallete = [
            "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white",
        ] 

    return int(str(pallete.index(colors[0])) + str(pallete.index(colors[1]))) 

File: csharp/resistor-color-duo/ResistorColorDuo.cs
using System;
using System.Linq;
using System.Collections.Generic;

public static class ResistorColorDuo
{
    private static List<string> pallette = new List<string>
    {
            "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white"
    };
    public static int Value(string[] colors)
    {
        return pallette.IndexOf(colors[0])*10 + pallette.IndexOf(colors[1]) ;
    }
}
